packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -182,6 +204,7 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 										chat.messages[ index + 1 ]?.role
 									) }
 									displayChatWithSupportLabel={ displayChatWithSupportLabel }
+									displayChatWithSupportEndedLabel={ displayChatWithSupportEndedLabel }
```

Comment:
Can we decide in `messages-container` what text to show in the label and only passing the text to ChatMessage?